src/config/mod.rs

```diff
@@ -1337,6 +1337,7 @@ mod pipelines_tests {
     use super::{load_from_str, Format};
 
     #[test]
+    #[ignore]
```

Comment:
Can we leave a note for why this is ignored for now? I like to see a reason why it's sticking around rather than being removed or tested.

Reply:
I don't know why I set this test to be ignored... I'll undo that and see if it runs or add a comment why it needs to be ignored, hehe.